docs/_ldio/ldio-transformers/ldio-version-object-creator.md

```diff
@@ -10,30 +10,78 @@ title: Version Object Creator
 
 The Version Object Creator will transform a State Object to a Version Object.
 
-## Config
+## State Object
+
+A state object is an entity that represents the most recent state of an object, and in this context, the subject is not unique. Below is an example presented in RDF format: Below is an example presented in RDF format:
 
-| Property               | Description                                                                                                                    | Required | Default           | Example                                   | Supported values |
-|:-----------------------|:-------------------------------------------------------------------------------------------------------------------------------|:---------|:------------------|:------------------------------------------|:-----------------|
-| date-observed-property | Property path (IRI format '<>') that points to a literal which should be used as timestampPath. Defaults to current timestamp. | No       | Current Timestamp | <https://example.org/ObservedAt>          | String           |
-| member-type            | Defines the RDF type of the version object                                                                                     | No       | N/A               | https://example.org/Person                | String           |
-| delimiter              | Defines how the version object id will be constructed. (versionOf + delimiter + dateObserved)                                  | No       | /                 | /                                         | String           |
-| generatedAt-property   | If defined, a statement will be added to the model with the observedAt value and the given property.                           | No       | N/A               | http://www.w3.org/ns/prov#generatedAtTime | String           |
-| versionOf-property     | If defined, a statement will be added to the model with the versionOf value and the given property.                            | No       | N/A               | http://purl.org/dc/terms/isVersionOf      | String           |
-
-A property path can be provided for the date-observed-property. 
-For example for the following ttl:
-```ttl
+```turtle
 @prefix time: <http://www.w3.org/2006/time#> .
 @prefix ex:   <http://example.org/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-ex:member
+<http://example.org/member>
```

Comment:
you could keep the prefix. As mentioned in the @prefix

Reply:
Jonas thought it would be a good idea to do it without the prefix to emphasize the different between the subject of a state object and a unique subject with a version object